app/src/views/private/components/drawer-item.vue

```diff
@@ -410,7 +410,11 @@ function useActions() {
 			internalEdits.value[primaryKeyField.value.field] = props.primaryKey;
 		}
 
-		emit('input', internalEdits.value);
+		const noNullDefaultValues = Object.fromEntries(
+			Object.entries(defaultValues.value).filter(([_, value]) => value !== null)
+		);
+
+		emit('input', merge({}, noNullDefaultValues, internalEdits.value));
```

Comment:
Should this be an `assign` instead of a `merge`? Otherwise you can erroneously end up merging into a object/array default value

Reply:
I couldn't find a way to get an `array` or `object` in the default values to break the `merge` but i do think that these could exist in the future with JSON fields for instance. Using `assign` seems more future proof to me 😄